src/main/webapp/style.css

```diff
@@ -1,8 +1,22 @@
+#emphasis {
+  color: violet;
+}
+
 h1 {
-  font-family: Verdana, sans-serif; 
-  font-size: x-large;
+  margin-top: 100px; 
 }
 
-#emphasis {
-  color: violet;
+.button {
```

Comment:
It might be good to give this a more specific name or to style the type button if this will be the default button look.

Reply:
Sounds good! I will make it a more specific id.